packages/adblocker-benchmarks/blockers/tsurlfilter.js

```diff
@@ -0,0 +1,55 @@
+/*!
+ * Copyright (c) 2017-present Cliqz GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+const {
+  Engine,
+  Request,
+  RequestType,
+  RuleStorage,
+  StringRuleList,
+  setConfiguration,
+} = require('@adguard/tsurlfilter/dist/bundle.min.cjs');
+
+const requestTypes = new Map([
+  [ 'sub_frame', RequestType.Subdocument ],
+  [ 'script', RequestType.Script ],
+  [ 'stylesheet', RequestType.Stylesheet ],
+  [ 'image', RequestType.Image ],
+  [ 'xmlhttprequest', RequestType.XmlHttpRequest ],
+  [ 'media', RequestType.Media ],
+  [ 'font', RequestType.Font ],
+  [ 'other', RequestType.Other ],
+]);
+
+module.exports = class TSUrlFilter {
+  static async initialize() {
```

Comment:
@@ameshkov the `initialize()` function is supposed to measure any global initialization for the library. I thought it was appropriate to call `setConfiguration()` here.